include/Plugin.h

```diff
@@ -142,13 +142,15 @@ class EXPORT Plugin : public Model, public JournallingObject
 	typedef QList<Descriptor> DescriptorList;
 
 	// contructor of a plugin
-	Plugin( const Descriptor* descriptor, Model* parent );
+	Plugin( const Descriptor * descriptor, Model * parent );
```

Comment:
As per recent discussions, I think the conclusion is to use either `type* var` or `type *var`.